examples/ssh/linux_updates.js

```diff
@@ -0,0 +1,123 @@
+/** Name: Linux Updates Monitoring
+ * Description: Monitors the status of the Updates of a Linux machine
+ *   
+ * Communication protocol is SSH, using the Linux bash command.
+ * 
+ * Tested on Linux with the following specs:
+ *      - Debian Linux 10
+ *      - bash version 5.0.3(1)
+ * 
+ * Requires:
+ *      - requires apt
+ *      - requires sed, grep, and awk
+ *      - PLEASE NOTE: it requires to be rut as root
+ * 
+ * Creates a Custom Driver Variable with the Number of Updates available
+ * 
+ * Creates a Custom Driver Table with the following columns:
+ *  - Pkg Name
+ *  - Old Version
+ *  - New Version 
+ * 
+**/
+
+var cmdNumberOfUpdates = "sudo apt update -qq 2>\/dev\/null | sed 's\/[^0-9 ]*\/\/g' > /tmp/updatesno.txt; cat /tmp/updatesno.txt"
```

Comment:
Why create a temporary file for a single number and then read it instead of just reading/parsing the number ?